src/EditorFeatures/CSharpTest/Diagnostics/GenerateConstructor/GenerateConstructorTests.cs

```diff
@@ -785,5 +784,83 @@ public bool M(string s, int i, bool b)
     }
 }");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateConstructor), Trait(Traits.Feature, Test.Utilities.Traits.Features.Tuples)]
+        public async Task Tuple()
+        {
+            await TestAsync(
+@"class C { void M() { new [|C|]((1, ""hello"")); } }",
+@"class C { private (int, string) p; public C((int, string) p) { this.p = p; } void M() { new C((1, ""hello"")); } }",
+parseOptions: TestOptions.Regular.WithTuplesFeature());
+        }
```

Comment:
may be test for a constructor that takes other types in addition to a tuple type? `new C((1, true), "hello")`